cli/server_enable_backup.go

```diff
@@ -19,9 +19,6 @@ func newServerEnableBackupCommand(cli *CLI) *cobra.Command {
 	cmd.Flags().String(
 		"window", "",
 		"The time window for the daily backup to run. All times are in UTC. 22-02 means that the backup will be started between 10 PM and 2 AM.")
```

Comment:
What about adding a deprecation notice to the flag’s usage description?

Reply:
I added a `(deprecated)` to the beginning of the sentence, because so you see it directly.